packages/substrate-codecs/test/AccountId.spec.ts

```diff
@@ -1,6 +1,7 @@
 import { expect, test } from "vitest"
 import { bytesToHex } from "@noble/hashes/utils"
 
+import { decodeBase58, encodeBase58 } from "../src/base58"
```

Comment:
Sorry, for some reason the final comment of the review didn't get published:

First and foremost: thanks a lot @kratico ! This is great work!

However, I would really rather use a predefined array with random bytes in the tests which shouldn't pass the checksum, rather than importing internal utils which are not part of the public API 🙏. Does that make sense?